qa-portal-angular/projects/portal-core/src/app/_common/services/qa-help.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { Router } from '@angular/router';
+import { HttpClient } from '@angular/common/http';
+import { HelpModel } from '../models/help-model';
+import { filter, map } from 'rxjs/operators';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class QaHelpService {
+
+  private messageSource = new BehaviorSubject("");
+  currentMessage = this.messageSource.asObservable();
+  private helpContent;
+  private userType;
+
+  constructor(private router: Router, private http: HttpClient) { }
+
+  getUserType(roleBool){
+    if (roleBool == true){
+      this.userType = "USER";
+    } else {
+      this.userType = "ADMIN";
+    }
+    return this.userType;
+  }
+
+
+  accessFile(): Observable <HelpModel>{
+    return this.http.get<HelpModel>('../assets/Textfiles/homeHelp.json', {responseType: 'json'});
+  }
+
+changeMessage(data, tag){
+  console.log(tag);
+  data.subscribe(data => this.helpContent = data.filter(help => help.Tag == tag || help.Tag + String(tag).match(/[0-9]+/) == tag));
```

Comment:
Please try to do all of these in a single subscribe.

Reply:
Cut down the observable to be one subscribe